File: 4344.cpp
#include <iostream>
#include <stdio.h>
using namespace std;

int main(){
	int number_of_testcase = 0, arr[1005], good_student[1005] = {0,}, sum = 0, number_of_people[1005] = {0, };
	double avg = 0.0, percentage[1005] = {0.0, };
	cin >> number_of_testcase;
	for(int i = 0 ; i < number_of_testcase ; i++){
		avg = 0.0, sum = 0;
		cin >> number_of_people[i];
		for(int j = 0 ; j < number_of_people[i] ; j++){
			cin >> arr[j];
			sum += arr[j];
		}
		avg = sum / number_of_people[i];
		for(int j = 0 ; j < number_of_people[i] ; j++){
			if(arr[j] > avg) good_student[i]++;
		}
		percentage[i] = (double)good_student[i] / (double)number_of_people[i];
	}
	for(int i = 0 ; i < number_of_testcase; i++){
		printf("%.3f%%\n", percentage[i] * 100);
	
	}

	return 0;
}

File: 4344.py
number_of_testcase = input()
number_of_testcase = int(number_of_testcase)
people_list = []
percentage_list = []
for i in range(number_of_testcase):
	summation = 0
	good_student = 0
	number_of_people = input()
	number_of_people = int(number_of_people)
	for j in range(number_of_people):
		score = input()
		score = int(score)
		summation = summation + score
		people_list.append(score)
	avg = summation / number_of_people
	for j in range(number_of_people):
		if people_list[j] > avg:
			good_student = good_student + 1

	percentage_list.append(good_student/number_of_people*100)

for value in percentage_list:
	print(str(value) + "%")